pkgs/applications/science/electronics/sv-lang/default.nix

```diff
@@ -55,14 +39,11 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    unordered_dense
     boost182
-    fmt_9
+    fmt_10
   ];
 
-  # TODO: a mysterious linker error occurs when building the unittests on darwin.
-  # The error occurs when using catch2_3 in nixpkgs, not when fetching catch2_3 using CMake
-  doCheck = !stdenv.isDarwin;
+  doCheck = true;
```

Comment:
Did you try Darwin?

Reply:
Actually, it was pre-emptive: I was optimistically hoping to let the CI system run it on my behalf so I could avoiding rebooting over from Asahi Linux into macOS. :) If it fails I'll take a peek and look into it or revert this, rest assured.